.github/workflows/coverity-results.yaml

```diff
@@ -0,0 +1,21 @@
+name: Coverity Results
+on:
+  workflow_dispatch:
+  push:
+  pull_request:
```

Comment:
This will get the report of defects on every push and pull-request. Is that what we want?

The issue is that the report would not be tied to the analysis of the actual commit. I guess we need to get the report following a Coverity run on the commit.

Reply:
Still in draft mode. Intention is that it should run on schedule, so will fix that once I can actually get a sensible report.

Reply:
This will go to the internal process repository instead. It does not need to be in this repository.